shell/platform/darwin/macos/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -167,4 +199,102 @@ - (bool)testFirstRectForCharacterRange {
   ASSERT_TRUE([[FlutterInputPluginTestObjc alloc] testFirstRectForCharacterRange]);
 }
 
+TEST(FlutterTextInputPluginTest, CanWorkWithFlutterTextField) {
+  FlutterEngine* engine = CreateTestEngine();
+  NSString* fixtures = @(testing::GetFixturesPath());
+  FlutterDartProject* project = [[FlutterDartProject alloc]
+      initWithAssetsPath:fixtures
+             ICUDataPath:[fixtures stringByAppendingString:@"/icudtl.dat"]];
+  FlutterViewController* viewController = [[FlutterViewController alloc] initWithProject:project];
+  [viewController loadView];
+  [engine setViewController:viewController];
+  // Creates a NSWindow so that the native text field can become first responder.
```

Comment:
nit: `s/Creates/Create/` since we're discussing what we do here, rather than documenting a function/method.